.pre-commit-config.yaml

```diff
@@ -67,3 +67,10 @@ repos:
         files: ^(telegram|examples|tests)/.*\.py$
         args:
           - --py37-plus
+-   repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+    -   id: isort
+        name: isort (python)
+        # compatibility with black
+        args: ["--profile", "black", "--filter-files", "-l", "-99"]
```

Comment:
what does `"--filter-files", "-l", "-99"` do?

why is it necessary to set the profile again here instead of just in the pyproject.toml?

Reply:
> [This](https://pycqa.github.io/isort/docs/configuration/options.html#filter-files) is what filter-files does.

I found that as well, but what does "filtering files" mean? 🤔 

> `-l` sets the line length. I'll test without this and see if it uses pyproject.toml

ah, it's ell, not one 🤦‍♂️

Reply:
> I found that as well, but what does "filtering files" mean? 🤔

tbh I don't know either, removing